joomla/joomla-cms
# [4.2] Allow typing in media field

Pull Request for Issue # . Replaces #36945

### Summary of Changes

- Removes the readonly attribute from the input of Media Field so users could type freely
- It doesn't fix the preview for EXTERNAL urls (out of scope)

### Testing Instructions
- Create a new article and add some title
- go to the images tab
- type (or copy paste) `images/sampledata/cassiopeia/not-valid.jpg`
- save the form
- type (or copy paste) `images/sampledata/cassiopeia/nasa1-400.jpg`
- Save the form (observe that the image url has the `#` parts!!!!)
- type (or copy paste) `/images/sampledata/cassiopeia/nasa1-400.jpg`
- save the form (observe the URL)
- type (or copy paste)  any valid external URL
- save the form 

![Screenshot 2022-04-25 at 16 31 44](https://user-images.githubusercontent.com/3889375/165117893-79e576f6-9d45-440e-aca0-538f383bcf01.png)

![Screenshot 2022-04-25 at 16 32 11](https://user-images.githubusercontent.com/3889375/165117924-2242d35f-59cd-4b15-8abf-c1c5d9b5c224.png)

![Screenshot 2022-04-25 at 16 32 22](https://user-images.githubusercontent.com/3889375/165117950-42cf2f0e-aeb9-4267-95f7-52b9604d5e80.png)


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
